bedrock/firefox/templates/firefox/developer.html

```diff
@@ -93,6 +93,28 @@
 <section class="feature-overview">
   <div class="container">
     <ul class="features">
+    {% if LANG.startswith('en-') %}
```

Comment:
These should be behind a tag, not en-US only. We'll want to localize them at some point after Nov 3rd.